content/articles/creating-a-simple-hunt-the-wumpus-game-using-python-flask-and-socketio/index.md

```diff
@@ -0,0 +1,943 @@
+        <meta name="description" content="In Hunt the Wumpus, the player is an adventurer entering a dungeon looking for a treasure. The goal of the adventurer is to find the treasure and get out of the dungeon avoiding the pits where he can fall in and the Wumpus: a monster who kills anyone it sees.">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+        <!--Fonts from Google Fonts-->
+        <!--Eczar-->
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Eczar:wght@400;500;600;700;800&display=swap" rel="stylesheet"> 
+
+        <!--CSS-->
+        <link rel= "stylesheet" type= "text/css" href= "{{ url_for('static',filename='styles/main.css') }}">
+
+        <!--JavaScript-->
+        <!--Socketio-->
+        <script src="https://cdn.socket.io/4.4.1/socket.io.min.js" integrity="sha384-fKnu0iswBIqkjxrhQCTZ7qlLHOFEgNkRmK2vaO/LbTZSXdJfAu6ewRBdwHPhBo/H" crossorigin="anonymous"></script>
+        <!--jQuery-->
+        <script src="https://code.jquery.com/jquery-3.6.0.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script>
+        <!--Feather Icons-->
+        <script src="https://cdn.jsdelivr.net/npm/feather-icons/dist/feather.min.js"></script>
+        <!--main.js-->
+        <script src="{{ url_for('static', filename='scripts/main.js') }}"></script>
+    </head>
+    <body>
+
+        <div class='header'>
+            <div class='title'>
+                <a href='#'><h2>Hunt the Wumpus</h2></a>
+            </div>
+        </div>
+
+        {% block content %}
+        {% endblock %}
+
+        <div class='bottom'>
+            <p>Created by <em>your name here</em>, 2022</p>
+        </div>
+
+    </body>
+</html>
+```
+
+Add some styles too in `main.css`:
+
+```CSS
+html, body {
+    margin: 0;
+    padding: 0;
+    font-family: 'Eczar', 'serif';
+    width: 100%;
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+    background-color: rgb(240, 220, 180);
+    color: rgb(100, 10, 10);
+}
+
+a {
+    color: rgb(100, 10, 10);
+    text-decoration-line: underline;
+    text-decoration-style: wavy;
+}
+
+h1, h2 {
+    line-height: 100%;
+}
+
+.header {
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+}
+
+.bottom {
+    margin-top: auto;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+}
+```
+
+Let's now extend the `base.html` in `index.html` and add a "Hello World!" to the page.
+
+```HTML
+{% extends 'base.html' %}
+
+{% block content %}
+
+<div class='main'>
+    <h1>Hello World!</h1>
+</div>
+
+{% endblock %}
+```
+
+Let's add some style to the main div in `main.css`:
+
+```CSS
+.main {
+    display: flex;
+    flex-direction: column;
```

Comment:
Do the same for other CSS snippets too. Unless you have something really important to explain